### DISCUSSION TITLE
스터디 목표 설정

### DISCUSSION THREAD

**Discussion by punchdrunkard** [prgrms-web-devcourse/We-are-the-MASTERS-OF-HTTP#1] [essay]
스터디를 통해 얻고 싶은 것, 기대하는 점 등등을 적어주세요!

**Comment by Hyevvy** [comment]
- 네트워크에 대해 좀 더 알아가며 실 업무에 적용해보고 싶습니다🙂
- 또, 오랜만에 개발자분들과 스터디를 하며 많은 열정을 얻어가고 싶습니다

다들 새해 복 많이 받으시고 스터디 끝까지 함께 해요 !

**Comment by punchdrunkard** [comment]
- 네트워크 관련 내용을 탄탄히 하고 싶습니다! 특히나 웹 개발에 있어서 가장 자주 사용되는 HTTP의 동작원리를 정확하게 알고, 사용하고 싶습니다. (axios 사용법 구글에 그만 검색하고 싶어요 ㅠㅠ)
- 책 한권을 단순히 완독하는 것보다 책의 내용을 저의 언어로 설명하고 싶습니다!

**Comment by yunjjeongjo** [comment]
- 조만간 백엔드 업무도 맡게 될 것 같은데 HTTP에 대해서 좀 더 알면 앞으로 새로 배울 분야에 많은 도움이 될 것 같습니다 !
- 책 내용을 독후감처럼 기록하기 보다 말로 뱉을 수 있는 정도가 되고 싶슴돠
화이팅 ,,,, !!!

**Comment by sa02045** [comment]
프론트엔드 개발자 역시 HTTP를 잘 알아야한다고 생각합니다! HTTP 관련 지식을 쌓고 내것으로 만들어보고싶어요
